modules/update-mirror-repository.adoc

```diff
@@ -6,6 +6,11 @@
 [id="update-mirror-repository-adm-release-mirror_{context}"]
 = Mirroring images using the oc adm release mirror command
 
+[IMPORTANT]
+====
+To avoid excessive memory usage by the OpenShift Update Service application, you must mirror release images to a separate repository as described in the following procedure.
```

Comment:
We also want to bring back [this command][1] (also visible [here][2], since linking to code in a file a pull request deletes seems unreliable) to accomplish that, when re-hydrating `oc adm release mirror ...` content on the mirrored registry.  #33841 has the original implementation of this behavior and a commit message with more context, in case that's helpful.

[1]: https://github.com/openshift/openshift-docs/pull/55448/files#diff-54a24d6e45fb689d33f3900efb4f025b75118d1298592d155afee45ba0cab897L156-L160
[2]: https://github.com/openshift/openshift-docs/blame/a9eea486ac20571635f3eda6a07eb3068c1de854/modules/update-service-mirror-release.adoc#L156-L160

Reply:
Mirroring to a separate repository is mandatory for OSUS only while the section works for both upgrading with OSUS and without OSUS. So I'd suggest we add a step in the end of the section, for example

```
[IMPORTANT]
====
If you want to update your cluster using the OpenShift Update Service application, you must mirror release images to a separate repository as described in the following procedure. Otherwise, you can skip the step
====
5. Mirror release images to a separate repository
$ oc image mirror -a ${LOCAL_SECRET_JSON} ${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}:${OCP_RELEASE}-${ARCHITECTURE} ${LOCAL_REGISTRY}/${LOCAL_RELEASE_IMAGES_REPOSITORY}:${OCP_RELEASE}-${ARCHITECTURE}
```

Reply:
@wking and @shellyyang1989 how about if I begin both of these added steps with a statement that it's only applicable to OSUS users?

for example: `5. If you are using the OpenShift Update Service, mirror release images to a separate repository`

I think this aligns with how we're supposed to write conditional steps per the style guide (that is, starting the instruction "if you <satisfy this condition>, do this step"

It can also be phrased like "OSUS users only: do this step"

Reply:
If you meant adding the statement to below steps, it's fine with me. But we still need to move the step `Mirror the release image to a separate repository` to the end of the section because the current last step `If the local container registry and the cluster are connected to the mirror host, directly push the release images to the local registry and apply the config map to the cluster by using following command:` is mirroring the manifests and release images to the same repository.

```
.. Export an additional local repository name to contain the release images:
+
[source,terminal]
----
$ LOCAL_RELEASE_IMAGES_REPOSITORY='<local_release_images_repository_name>'
----
+
For `<local_release_images_repository_name>`, specify the name of the repository to
create in your registry, such as `ocp4/openshift4-release-images`.
```
and

```
... Mirror the release image to a separate repository:
+
[source,terminal]
----
$ oc image mirror -a ${LOCAL_SECRET_JSON} ${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}:${OCP_RELEASE}-${ARCHITECTURE} ${LOCAL_REGISTRY}/${LOCAL_RELEASE_IMAGES_REPOSITORY}:${OCP_RELEASE}-${ARCHITECTURE}
```

Reply:
@shellyyang1989 Okay thank you for clarifying, I understand what you mean. Sorry for any confusion!

Could you take a look at the procedure again to see if I made the changes you were imagining for that final step you mentioned?